.swiftlint.yml

```diff
@@ -35,7 +35,7 @@ identifier_name:
         - no
         - ok
 line_length:
-    warning: 150
+    warning: 180
```

Comment:
the 150 limit can easily be reached on log lines, especially if we are using LOG TAG from another class, e.g. `EdgeConstants.LOG_TAG`